x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/findings_by_resource_table.test.tsx

```diff
@@ -22,12 +22,14 @@ const getFakeFindingsByResource = (): FindingsByResourcePage => {
   const total = chance.integer() + count + 1;
   const normalized = count / total;
 
+  const [resourceName, resourceSubtype, ...cisSections] = chance.unique(chance.word, 4);
```

Comment:
it's ok that these are unique within the `row` because we use `within(row)` in the test